app/lib/email-helper.js

```diff
@@ -11,7 +11,11 @@ const buildInBreach = data => {
     details = `Breach reported for Dog ${data?.dogChosen?.indexNumber}\n`
   }
   details += 'Reasons:\n'
-  details += data?.dogBreaches?.map(breach => ` - ${breach}\n`).join('')
+  if (!data?.dogBreaches) {
+    details += `${data?.details}\n`
+  } else {
+    details += data?.dogBreaches?.map(breach => ` - ${breach}\n`).join('')
+  }
```

Comment:
pretty sure you'll need some more coverage with all the optional chaining

Reply:
I added an extra test to cover it